utils.rs

```diff
@@ -0,0 +1,19 @@
+use std::io::{Error, ErrorKind, Result};
```

Comment:
Please add a license header to this. (You can copy it from any existing `.rs` file.)